phpunit/class-do-blocks-test.php

```diff
@@ -22,4 +22,26 @@ function test_do_blocks_removes_comments() {
 
 		$this->assertEquals( $expected_html, $actual_html );
 	}
+
+	/**
+	 * Test that shortcode blocks get the same HTML as shortcodes in Classic content.
+	 */
+	function test_the_content() {
+		add_shortcode( 'someshortcode', array( $this, 'handle_shortcode' ) );
```

Comment:
I'm not as familiar with PHP testing, but should we be tearing down anything we create, i.e. a complementing `remove_shortcode`?